lib/footnotes.js

```diff
@@ -24,24 +24,26 @@ function setRef(link)
 
 function processPage(page)
 {
-  var options = this.config.options
+  if(this.output.name !== 'ebook') return page
 
-  if(options.generator !== 'pdf') return page
+  var i10nPrefix = intl[this.config.get('language')] || intl['en']
+  let pageLinks = getPageLinks(page, getMaxFootnote(page.content))
 
-  var language = intl[options.language || 'en']
+  if (!pageLinks.length) return page
 
-  getPageLinks(page, getMaxFootnote(page.content))
-  .map(setRef, page)
-  .forEach(function createFootnote(link)
-  {
-    var linkUrl = link.url
+  page.content += '\n\n---\n\n'
```

Comment:
i tried it, and it's currently not possible via css, because both quotes as well as footnotes use the same markup (`<blockquote>`), see https://github.com/GitbookIO/markup-it/issues/20#issuecomment-288811053